embed.fnc

```diff
@@ -4388,6 +4388,14 @@ S	|const char *|my_langinfo_i					\
 				|NULLOK Size_t *retbuf_sizep		\
```

Comment:
commit message:

Is this even valid?
```
export LC_ALL="LC_CTYPE=de_DE;LC_NUMERIC=C;LC_TIME=de_DE;..."
```
If I try with the glibc setlocale on Debian stable:
```
tony@venus:.../perl/git$ cat locale-disp.c
#include <locale.h>
#include <stdio.h>

int main() {
    printf("setlocale %s\n", setlocale(LC_ALL, ""));
}
tony@venus:.../perl/git$ gcc -olocale-disp locale-disp.c 
tony@venus:.../perl/git$ ./locale-disp 
setlocale en_AU.UTF-8
tony@venus:.../perl/git$ LC_MESSAGES=de_DE.utf8 ./locale-disp 
setlocale LC_CTYPE=en_AU.UTF-8;LC_NUMERIC=en_AU.UTF-8;LC_TIME=en_AU.UTF-8;LC_COLLATE=en_AU.UTF-8;LC_MONETARY=en_AU.UTF-8;LC_MESSAGES=de_DE.utf8;LC_PAPER=en_AU.UTF-8;LC_NAME=en_AU.UTF-8;LC_ADDRESS=en_AU.UTF-8;LC_TELEPHONE=en_AU.UTF-8;LC_MEASUREMENT=en_AU.UTF-8;LC_IDENTIFICATION=en_AU.UTF-8
tony@venus:.../perl/git$ LC_ALL="en_AU.UTF-8;LC_NUMERIC=en_AU.UTF-8;LC_TIME=en_AU.UTF-8;LC_COLLATE=en_AU.UTF-8;LC_MONETARY=en_AU.UTF-8;LC_MESSAGES=de_DE.utf8;LC_PAPER=en_AU.UTF-8;LC_NAME=en_AU.UTF-8;LC_ADDRESS=en_AU.UTF-8;LC_TELEPHONE=en_AU.UTF-8;LC_MEASUREMENT=en_AU.UTF-8;LC_IDENTIFICATION=en_AU.UTF-8" ./locale-disp 
setlocale (null)
```

Reply:
Try s/utf8/UTF-8/.  That works for me